docs/getting_started.md

```diff
@@ -216,6 +219,92 @@ Here are some resources that can be used to get you started working with `AnnDat
+Cells are labeled either as `"Keep"` (cells to retain) or `"Remove"` (cells to filter out).
+
+To filter cells based on this column, use the following command:
+```r
+# Filter cells based on ADT QC statistics
+processed_sce <- processed_sce[, which(processed_sce$adt_scpca_filter == "Keep")]
+```
+
+It is also important to be aware that the normalized ADT expression matrix only contains values for cells labeled as `"Keep"` in the `adt_scpca_filter` column.
+Any cells labeled `"Remove"` have `NA` values normalized expression matrix (see
+{ref}`processed cite-seq data <processing_information:Processed CITE-seq data>` for more details).
+
+As dimension reduction requires all known values (i.e., no `NA`s), filtering cells as shown above is critical before proceeding to downstream analyses on ADT counts.
+
+To perform principal components analysis (PCA) on the ADT counts, use the following `scater` command _after_ filtering has been performed:
+
+```r
+# Calculate PCA on the ADT counts
+processed_sce <- scater::runPCA(altExp(processed_sce))
+```
+
+To perform UMAP on this resulting PCA matrix, use the following `scater` command:
+
+```r
+# Calculate UMAP on the ADT counts
+processed_sce <- scater::runUMAP(altExp(processed_sce))
+```
+### Filtering cells based on ADT quality control
+
+In both the `filtered.rds` and `processed.rds` files, cells are flagged for removal based on ADT-level QC statistics in the alternative experiment's `discard` column as calculated by [`DropletUtils::CleanTagCounts()](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html).
+This column contains values `FALSE` for cells that should not be removed and `TRUE` for cells that should be removed.
+
+To filter cells based on ADT counts, for example in the `filtered.rds` object, use the following commands:
+
+```r
+# First, you can see cells should be removed (`TRUE`) vs. kept (`FALSE`)
+altExp(filtered_sce)$discard
+
+# Filter cells based on ADT QC statistics
+filtered_sce <- filtered_sce[, -which(altExp(filtered_sce)$discard)]
+```
+
+### Normalizing ADT counts
+
+The `filtered.rds` object contains a raw ADT counts matrix, but not a matrix of log normalized ADT expression.
+To perform normalization yourself, we first recommend filtering cells using the `filtered_sce$discard` column, as shown above, to reduce the chances of normalization failing as described in
+{ref}`processed cite-seq data <processing_information:Processed CITE-seq data>`.
+
+Normalization can then be performed with `scuttle` and `scater`:
+
+```r
+# Calculate size factors for use in ADT normalization,
+# using the previously-calculated ambient profile
+altExp(filtered_sce) <- scuttle::computeMedianFactors(
+  altExp(filtered_sce),
+  reference = metadata(altExp(filtered_sce))$ambient_profile
+)
+
+# Normalize and log transform
+# This will only succeed if all size factors calculated above are positive
+altExp(filtered_sce) <- scater::logNormCounts(altExp(filtered_sce))
+```
+
```

Comment:
Maybe just a statement saying you can now proceed with dimensionality reduction as above?

Reply:
done in https://github.com/AlexsLemonade/scpca-docs/pull/115/commits/9ae14dea04d52044798f3073511855bb71c8ba04